commit db19be6b84c89b8dbcbe6abe85a1c1daea384100
Author: Andrew Hsieh <andrewhsieh@google.com>
Date:   Wed Apr 3 14:09:57 2013 +0800

    Misc fixes
    
    1. Fixed darwin build of GCC 4.8/X86 by removing trailing
       ',' from the last item of enum.  Not discovered before
       because GCC 4.8 now builds with -Wall -Werror, and gcc-4.2.1
       in darwin is picky about that
    2. Fixed -fuse-ld=mcld
    
    Change-Id: I7b65edfb76841f14c3290acb5a98f556d00e1139

diff --git a/gcc-4.7/gcc/config/i386/i386-opts.h b/gcc-4.7/gcc/config/i386/i386-opts.h
index 8904491..4f92987 100644
--- a/gcc-4.7/gcc/config/i386/i386-opts.h
+++ b/gcc-4.7/gcc/config/i386/i386-opts.h
@@ -84,7 +84,7 @@ enum ix86_veclibabi {
 
 enum stack_protector_guard {
   SSP_TLS,      /* per-thread canary at %gs:20 */
-  SSP_GLOBAL,   /* global canary */
+  SSP_GLOBAL    /* global canary */
 };
 
 #endif
